mathesar/imports/csv.py

```diff
@@ -28,4 +27,5 @@ def create_table_from_csv(name, schema, database_key, csv_file):
     database = get_database_key(engine)
     schema, _ = Schema.objects.get_or_create(name=db_table.schema, database=database)
     table, _ = Table.objects.get_or_create(name=db_table.name, schema=schema)
+    table.create_records([row for row in csv_reader])
```

Comment:
Given my comment about the connections on the `create_records` function, this will be pretty slow for large CSVs.  We'll need to update the function to use the `COPY` function (probably via the `copy_expert` function of `psycopg2`) at some point.

Reply:
We'll only be doing `get_record` if there's a single record being created. I renamed the `create_records` function, restructured it a bit to combine connections, and added some comments to clarify.